README.md

```diff
@@ -817,6 +817,12 @@ All statements above will return false if used with `===`
 
 🔗 [**Read More: Common security best practices**](/sections/security/commonsecuritybestpractices.md)
 
+## ![✔] 6.15. Use cookies/sessions securely
```

Comment:
Title should be related to Node.JS (not generic security advice). 

For example: Modify the default session middlewares settings